packages/site/src/components/BodyBlock.tsx

```diff
@@ -14,7 +14,13 @@ export const BodyBlock = ({ children, title }: BodyBlockProps) => {
   return (
     <Box padding={"extravagant"}>
       <Content>
-        <Typography size="larger" fontWeight="bold">
+        <Typography
```

Comment:
Match styling from searchbox

This will be increasingly useful once we have component groups implemented